tests/data/placebo/test_sqs_add_statements_policy/sqs.GetQueueAttributes_10.json

```diff
@@ -0,0 +1,29 @@
+{
+    "status_code": 200,
+    "data": {
+        "Attributes": {
+            "QueueArn": "arn:aws:sqs:us-east-1:644160558196:test-queue",
```

Comment:
It would be good to also replace the accountids with something safe like 111222333444 or 123456789123 to help obscure the accountid that you are actually working out of